### ISSUE TITLE
Map Block: Unauthorized: you may have provided an invalid Mapbox access token.

### ISSUE THREAD

**Issue by ktyfuller604** [Automattic/wp-calypso#39783]
#### Steps to reproduce
1. Starting at URL: https://realestater.agency/
2. Edit any page, or create a new page, in Calypso
3. Add a Map Block

#### What I expected

The map block would show a map allowing me to add a pin. 

#### What happened instead

Unauthorized: you may have provided an invalid Mapbox access token. See https://www.mapbox.com/api-documentation/#access-tokens-and-token-scopes

#### Additional Testing
* There are two sites on the same account, I tested with the second site and did not get the same error
* I tested on my own site and did not get the error
* Noticed on that specific site there's a field to update the API key, but that option doesn't show on my other tests
* It appears to only happen on this specific site https://realestater.agency/

#### Screenshot / Video

![Screenshot](https://d.pr/i/3VJbru+)
Image URL: https://d.pr/i/3VJbru

#user-report
2752229-t

**Comment by simison** [comment]
cc @Automattic/explorers

**Comment by Copons** [comment]
Hi @ktyfuller604 thanks for reporting!
I gave it a try and I believe that the error message is telling the truth. 🙂 

We recently shipped an update to the Map block to make it work _without_ a personal access token on Simple and Atomic sites.
Seeing that the site there uses a personal token, my first attempt was simply to remove it.
As intended, without personal token, the Map block was indeed working.
(FYI, their other site is not using any tokens, reason why the block is working fine there.)

But, since WP.com users can always provide their token if they want, the Map block was still supposed to work with their token.
So I've also tried with my own token, and the Map block worked fine.

tl;dr I think the access token they are using is incorrect.
You can invite them to double check their personal token and update it from the Map block sidebar, or you can simply tell them to remove it (from the same place).
The Mapbox access token is shared across all Map blocks on the same site, so they only need to update or remove it once.

Just one thing in case it comes up with the user: this action, while it affects all Map blocks, only visually updates "in real time" the currently selected block.
This means that if they have multiple Map blocks in the same editor (or, say, other editors with Map blocks open in other tabs), after updating the token they will need to reload the page to see the effect on all blocks.
